### DISCUSSION TITLE
Add an Exclude folders config field

### DISCUSSION THREAD

**Discussion by dna1000** [glowingjade/obsidian-smart-composer#128] [Ideas & Feature Requests]
Firstly, Greate plugin, extremely useful.

For my use case in the configuration, I have a config folder with lots of sub-folders which I use to hold various plugin configs (to avoid having lots of top-level config folders).
It would be nice to have a field to enter exclude folders in addition to the exclude patterns for all folder and sub-folder etc.

ie, in my case I have to set the patterns:
Config_Folder/PluginA/subfolder1/*
Config_Folder/PluginA/subfolder2/*
Config_Folder/PluginA/*
Config_Folder/PluginB/subfolder1/*
Config_Folder/PluginB/subfolder2/*
Config_Folder/PluginB/*
Config_Folder/*

It would be nice to just be able to enter 'Config_Folder' and everything inside is excluded.

(I have tried a few different pattern globs to try and exclude inside a folder but the only way that works is to explicitly define each folder/subdir/subdir/*)

Cheers

**Comment by kevin-on** [comment]
Hi, thanks for the comment.
Maybe`Config_Folder/**` will work in your case.

**Reply by dna1000** [reply]
Flippin heck. I tried every which way but not `folder/**`.
That worked a treat! Thanks

**Reply by glowingjade** [reply]
I've also added these tips to the wiki:
https://github.com/glowingjade/obsidian-smart-composer/wiki/1.4-Advanced-Configuration#rag-settings
I'll close this discussion as it seems to be resolved.
